relayer/src/message_relayer/common/gear/message_sender/compose_payload.rs

```diff
@@ -98,6 +118,20 @@ async fn build_inclusion_proof(
         .await?;
 
     let slot = beacon_block.slot;
+    let endpoint = historical_proxy
+        .endpoint_for(slot)
+        .recv(historical_proxy_id)
+        .await
+        .map_err(|e| anyhow::anyhow!(e))?
+        .map_err(|e| anyhow::anyhow!("Proxy faield to get endpoint for slot #{}: {:?}", slot, e))?;
+
+    let (slot, checkpoint) = service_checkpoint
+        .get(slot)
+        .recv(endpoint)
```

Comment:
If i'm not mistaken the endpoint is `eth-events-*` instance not a `checkpoint-light-client` one